README.md

```diff
@@ -114,11 +114,28 @@ shell alias to check your documentation changes build correctly.
 alias build-docs='RUSTDOCFLAGS="--cfg docsrs" cargo +nightly rustdoc --features="$FEATURES" -- -D rustdoc::broken-intra-doc-links'
 ```
 
+## Testing
+
+### Unit/Integration tests
+
+Run as for any other Rust project `cargo test --all-features`.
+
 ### Running benchmarks
 
 We use a custom Rust compiler configuration conditional to guard the bench mark code. To run the
 bench marks use: `RUSTFLAGS='--cfg=bench' cargo +nightly bench`.
 
+### Running mutation tests
+
+We do mutation testing with [mutagen](https://github.com/llogiq/mutagen). To run these tests first
+install `mutagen` with `cargo +nightly install cargo-mutagen` then run with:
```

Comment:
From the mutagen README.
>  If you referenced mutagen in your cargo.toml via the git repository as noted in the Using Mutagen section, you will probably want to install the development version of cargo-mutagen. To install the development version, run cargo install in the mutagen-runner dir of this repository. Running cargo install --force might be necessary to overwrite any existing cargo-mutagen binary.


This worked for me and perhaps should also work for @apoelstra

Reply:
Had the same problem, this makes it work :+1: 

```
git clone https://github.com/llogiq/mutagen
cd mutagen/mutagen-runner/
cargo +nightly install --force --path .
```

Reply:
Shouldn't just `cargo +nightly install --git https://github.com/llogiq/mutagen` work?

Reply:
Thanks! Worked for me.

Reply:
Ahh, thanks @sanket1729, that is also how I installed it but I forgot.